apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { Test, TestSection, PlatformStatus } from '../Test';
+
+// const NotificationVariant string: 'primary' | 'neutral' | 'danger' | 'warning';
+enum NotificationVariant { primary = 1, neutral, danger, warning };
```

Comment:
Can we actually use the union over the enum? Some info on why unions are preferred:

https://blog.bam.tech/developer-news/should-you-use-enums-or-union-types-in-typescript

https://stackoverflow.com/questions/40275832/typescript-has-unions-so-are-enums-redundant

Reply:
I think you just need to declare it as a type. 

From `pacakges/experimental/Avatar/src/Avatar.tsx`: 
``` 
export type Size = 'xSmall' | 'small' | 'medium' | 'large' | 'xLarge' | 'xxLarge';
```
But also, now that I understand it better, I like what `packages/experimental/Badge/src/Badge.types` does too:

```
export const BadgeSizes = ['smallest', 'smaller', 'small', 'medium', 'large', 'largest'] as const;
export type BadgeSize = typeof BadgeSizes[number];
```